frontend/src/components/WofIcon.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div>
+        <i :class="icon" :style="`font-size: ${size}px`"></i>
```

Comment:
Also - in [all the examples I have seen](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1) the style bindings were done like this:
```suggestion
        <i :class="icon" :style="{ font-size: size + 'px' }"></i>
```
Is there a reason we shy away from it?

Reply:
It's kinda a bad practice in JS

Reply:
Why?